packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
+import ExampleVideo from '@site/src/components/ExampleVideo';
 
-The document explains how can you animate all layout changes for a specific view just by adding a single property to the view.
-To be precise how to animate positions and dimensions of components. What's important it will all happen entirely on UI thread without any communication through the bridge. There are plenty of ways in which you can animated layout changes however in contrast to entering and exiting animations they are not so regular. We've prepared a few predefined layout transitions, however if you want to create more custom and complex transition you can create your own.
+<InteractiveExample src={LayoutTransitionsSrc} component={LayoutTransitions} />
 
-## How to use predefined layout transition?
+## Predefined Transitions
+
+Below we listed all of the currently available predefined layout transitions with their [layout animation modifiers](/docs/fundamentals/glossary#layout-animation-modifier).
+
+## Linear transition
 
-### 1. Import chosen transition
+Linear transition animates both position and dimension in the same way.
+
+### Reference
 
 ```js
-// Transition is just an example and should be replaced by real animation. For Instance Layout
-import { Transition } from 'react-native-reanimated';
+import { LinearTransition } from 'react-native-reanimated';
+
+function App() {
+  return <Animated.View layout={LinearTransition} />;
+}
 ```
 
-### 2. Choose Animated Component which layout you want to animate
+### Modifiers
 
-```js
-    // AnimatedComponent - component created by createAnimatedComponent or imported from Reanimated
-    <AnimatedComponent layout={Transition} >
+#### Time-based <Optional/>
+
+Time-based modifiers relay on [`withTiming`](/docs/animations/withTiming) function.
+
+```javascript
+LinearTransition.duration(500).easing(Easing.ease);
 ```
 
-### 3. Customize the animation
+- `.easing(easingFunction: EasingFunction)` is an easing function which defines the animation curve. Defaults to `Easing.inOut(Easing.quad)`
+
+:::note
+Time-based modifiers have no effect when `.springify()` is used.
+:::
+
+#### Spring-based <Optional/>
+
+Spring-based modifiers relay on [`withSpring`](/docs/animations/withSpring) function.
+
+```javascript
+LinearTransition.springify()
+  .damping(30)
+  .mass(5)
+  .rotate(20)
+  .stiffness(10)
+  .overshootClamping(false)
+  .restDisplacementThreshold(0.1)
+  .restSpeedThreshold(5);
+```
+
+- `.springify()` enables the spring-based animation configuration.
+- `.damping(value: number)` decides how quickly a spring stops moving. Higher damping means the spring will come to rest faster. Defaults to `10`.
+- `.dampingRatio(value: number)` decides how damped the spring is. Value `1` means the spring is critically dmaped, and value `>1` means the spring is overdamped. Defaults to `0.5`.
+- `.mass(value: number)` is the weight of the spring. Reducing this value makes the animation faster. Defaults to `1`.
```

Comment:
There is no info, what is default - time or physics based. but perhaps there is no need for that.

Reply:
I will add it for better info :like: